uint16 ID
string cmd
---
string robot_state